src/modules/assertions/require_inline_coding.ts

```diff
@@ -0,0 +1,17 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+export const  requireInlineCoding = () => {
+  
+  const solfileRegex = /http:\/\/\S+.sol/g;
+  const prBodyString = (context.payload?.pull_request?.body != null) ? JSON.stringify(context.payload?.pull_request?.body) : null ;
```

Comment:
You should always use `===` and `!==` in JavaScript to avoid truthy/falsy comparisons.
```suggestion
  const prBodyString = (context.payload?.pull_request?.body !== null) ? JSON.stringify(context.payload?.pull_request?.body) : null ;
```